en/SUMMARY.md

```diff
@@ -168,22 +158,22 @@
   * [Full Parameter Reference](advanced_config/parameter_reference.md)
   * [ECL/EKF Overview & Tuning](advanced_config/tuning_the_ecl_ekf.md)
   * [Flight Termination Configuration](advanced_config/flight_termination.md)
-  * [Bootloader Flashing onto Betaflight Systems](advanced_config/bootloader_update_from_betaflight.md)
   * [Land Detector Configuration](advanced_config/land_detector.md)
   * [Prearm/Arm/Disarm Configuration](advanced_config/prearm_arm_disarm.md)
-  * [IMU Factory Calibration](advanced_config/imu_factory_calibration.md)
-  * [Sensor Thermal Compensation](advanced_config/sensor_thermal_calibration.md)
-  * [Compass Power Compensation](advanced_config/compass_power_compensation.md)
-  * [Advanced Controller Orientation](advanced_config/advanced_flight_controller_orientation_leveling.md)
   * [Static Pressure Buildup](advanced_config/static_pressure_buildup.md)
   * [Serial Port Configuration](peripherals/serial_configuration.md)
-  * [MAVLink Telemetry (OSD/GCS)](peripherals/mavlink_peripherals.md)
   * [PX4 Ethernet Setup](advanced_config/ethernet_setup.md)
-  * [Bootloader Update](advanced_config/bootloader_update.md)
-    * [Bootloader Update FMUv6X-RT via USB](advanced_config/bootloader_update_v6xrt.md)
-* [Hardware (Drones&Parts)](hardware/drone_parts.md)
-  * [Complete Vehicles](complete_vehicles/README.md)
+
+* [Hardware Selection & Setup](hardware/drone_parts.md)
   * [Flight Controllers (Autopilots)](flight_controller/README.md)
+    * [Mounting the Flight Controller](assembly/mount_and_orient_controller.md)
+    * [Updating Firmware](config/firmware.md)
+    * [Flight Controller/Sensor Orientation](config/flight_controller_orientation.md)
```

Comment:
FYI, note orientation would ideally be after "mounting". However it is here, because it is config, which must be done after firmware update.